streams/src/test/java/org/apache/kafka/streams/integration/EosBetaUpgradeIntegrationTest.java

```diff
@@ -906,29 +912,22 @@ public void close() { }
         return streams;
     }
 
-    private void waitForStateTransition(final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> observed,
-                                        final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> expected)
-            throws Exception {
-
+    private void waitForRunning(final List<KeyValue<KafkaStreams.State, KafkaStreams.State>> observed) throws Exception {
```

Comment:
This one actually might not be necessary, but I thought it was useful as a sanity check especially since we're trying to access the stores in some places

Reply:
Agreed.